sdk/iot/Azure.Iot.Hub.Service/src/ModulesClient.cs

```diff
@@ -184,7 +193,7 @@ public virtual Task<Response<BulkRegistryOperationResponse>> CreateIdentitiesWit
                     ImportMode = ExportImportDeviceImportMode.Create
                 });
 
-            return _registryManagerClient.BulkDeviceCrudAsync(registryOperations, cancellationToken);
+            return _devicesRestClient.BulkRegistryOperationsAsync(registryOperations, cancellationToken);
```

Comment:
This just uses the deviceRestClient. Any thoughts on getting the Bulk operations in a separate client instead of being part of DeviceRestClient?

Reply:
Our public API has all these operations under a single `DevicesClient` and `ModulesClient`, so I don't mind the internal generated code being housed under the same rest client as well.